### DISCUSSION TITLE
[firebase_analytics] Measure Google Ads conversions from my iOS app campaigns

### DISCUSSION THREAD

**Discussion by Lazizbek97** [firebase/flutterfire#10990] [Q&A]
I am using Google Ads campaigns for my IOS/Android app promotion. 
I used firebase for user auth(only with phone number). 
Now, i want to check/measure users interactions(like: app installs via ads, first open, etc.). For android it is showing fine on google ads console. But i cannot check statistics for ios.

I read documentation [here](https://firebase.google.com/docs/tutorials/ads-ios-on-device-measurement).
But it is showing only one method for swift: `Analytics.initiateOnDeviceConversionMeasurement(emailAddress: "example@gmail.com")`

How can i track users app installs event, first open event, etc. for my Flutter app? is there any method, or way to make it?

Thank you

**Comment by ktiniatros** [comment]
I'm looking for the same. Is there any way to measure flutter iOS app installs from google ads?

**Comment by daiki1003** [comment]
anything update?

**Comment by santa112358** [comment]
I'm facing the same issue. Has there been any progress or solution to this?
Would appreciate any updates or workarounds.

**Comment by masfour7** [comment]
Same issue. Any solution? Would appreciate it if you could share.

@Lazizbek97 @ktiniatros @daiki1003 @santa112358 
-Google Ads conversions from iOS app campaigns-

**Comment by Oliver-Zimmerman** [comment]
Same issue. Has anyone resolved this?

**Comment by lyqandy** [comment]
same issue

**Comment by alaudiero** [comment]
same issue
